AidenRheem.md

```diff
@@ -0,0 +1,23 @@
+---
+name: "@AidenRheem"
+project: "3D Printed Keyboard"
+---
+
+
+## Summary
+I'm going to be designing a custom 60% keyboard but using a open-source version called the Bakeneko60! I will also be creating a custom firmware for my keyboard off of a fork of an existing program called QMK. The keyboard will be used for everything I do (home & school) on my setup. I'm excited to build this because it's always been my goal to build a mechanical keyboard and modify it's firmware to make it my own. It'll be my first time building a keyboard as well. I've also never done C programming and I feel this project can help introduce me to it. As for what I am going to do for the firmware, I will customize it to include custom macros (shortcuts), layouts, and more. Hopefully this will be a experience for me to learn C programming, and building own keyboards and tinkering with electronics/modding them!
+
+## Plan
+For the first 2 days, i'll work on creating my keyboard using the Bakeneko60 that I purchased. For the next 6 days, i'll study the C programming language and familiarize myself with QMK firmware. I'll also look at several guides online about customizing the QMK software and developing it from scratch. I will also start developing the custom software. The remaining 2 days will be for me to finish up any software code that I have remaining, and making sure everything works, then placing it on my setup!
+
+
+## Budget
+Materials
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Case/PCB (contains most materials I need) | [Bakeneko60](https://cannonkeys.com/products/bakeneko-60?variant=40911986720879) | $134.99 |
+| Switches        | [Gateron Oil King](https://divinikey.com/products/gateron-oil-king-linear-switches) | $54.35 |
+| Keycaps         | [Minimalistic BoW](https://idobao.net/collections/keycap/products/idobao-minimalist-black-and-white-pbt-keycaps-kit) | $46.50 |
+| Puller Tool     | [Keycap & Switch Puller](https://www.amazon.com/dp/B09L59DNH2) | $4.99 |
+| Shipping        | Divinikey (Switches)                  | $6.01  |  
```

Comment:
hii :) is this shipping or switches?

Reply:
thats going to be shipping for the switches, sorry for the confusion